apps/docs/src/app/core/component-docs/timeline/timeline-docs.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+import * as timelineBasicExampleTs from '!raw-loader!./examples/timeline-basic-example/timeline-basic-example.component.ts';
+import * as timelineBasicDataExampleTs from '!raw-loader!./examples/timeline-basic-example/timeline-example-data';
+import * as timelineBasicExampleHtml from '!raw-loader!./examples/timeline-basic-example/timeline-basic-example.component.html';
+
+@Component({
+  selector: 'fd-timeline-docs',
+  templateUrl: './timeline-docs.component.html',
+})
+export class TimelineDocsComponent {
+
+  timelineBasicExample: ExampleFile[] = [
+    {
+      language: 'html',
+      code: timelineBasicExampleHtml,
+      fileName: 'timeline-basic-example'
+    },
+    {
+      language: 'typescript',
```

Comment:
Please mark with appropriate names :)

![image](https://user-images.githubusercontent.com/20265336/131392892-98f2acf8-0444-4201-9d12-01a03ad36983.png)

Reply:
Actually it strange to pass file name to field called `language` .
ExampleFile interface has field called fileName and I expect that it will be using, but it doesn't work. 
May be it would be better to fix this behavior?

Reply:
Sorry, I've picked not the exact line. You can specify a name

```
{
  language: 'typescript',
  code: platformTableDefaultTsSrc,
  fileName: 'platform-table-default-example',
  component: 'PlatformTableDefaultExampleComponent',
  name: 'platform-table-example.component.ts' // <= Here
}
```

Reply:
I got it)